platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less

```diff
@@ -1235,60 +1235,90 @@
   #activityComposerApp {
     .drawer {
       .content {
-        .attachmentsList{
-          display: flex;
-          .attachedFiles{
-            margin-left: 3%;
-          }
-          .uiIconAttach {
-            color: #333;
-          }
-        }
-        .composerActions {
-          margin-top: 8%;
-          padding-top: 8%;
-          border-top: 1px solid #efefef;
-
-          .actionItem {
+        .VuetifyApp {
+          .attachmentsList {
             display: flex;
             align-items: center;
-            padding: 10px 10px 0 10px;
-            border: 1px solid #E1E8EE;
-            margin: 10px 0;
-            border-radius: 5px;
-
-            .actionItemIcon {
-              .addFileComposerIcon {
-                background: url('/eXoSkin/skin/images/system/composer/addFile.png') no-repeat;
-                width: 60px;
-                height: 60px;
-                margin: 10px 0 0 10px;
-                background-size: 40px;
-              }
+
+            .attachedFiles {
+              margin-left: 3%;
+            }
+
+            .uiIconAttach {
+              top: -1px;
+              color: #333;
             }
 
-            .actionItemDescription {
-              padding: 9px 10px;
+            .v-progress-circular {
+              svg {
+                color: transparent;
+              }
 
-              .actionLabel {
-                font-weight: bold;
-                font-size: 16px;
+              circle {
+                color: #FFF;
               }
 
-              .actionDescription > p {
-                color: #A8B3C5;
-                font-size: 12px;
+              &.uploading {
+                svg {
+                  color: #578dc9;
+                }
+
+                circle.v-progress-circular__overlay {
+                  color: #578dc9;
+                }
               }
             }
+          }
+        }
 
-            &:hover {
-              box-shadow: 0 0 8px #888888;
-              cursor: pointer;
+        .v-navigation-drawer {
+          z-index: 20;
+        }
+      }
+
+      .composerActions {
+        margin-top: 8%;
+        padding-top: 8%;
+        border-top: 1px solid #efefef;
+
+        .actionItem {
+          display: flex;
+          align-items: center;
+          padding: 10px 10px 0 10px;
+          border: 1px solid #E1E8EE;
+          margin: 10px 0;
+          border-radius: 5px;
+
+          .actionItemIcon {
+            .addFileComposerIcon {
+              background: url('/eXoSkin/skin/images/system/composer/addFile.png') no-repeat;
+              width: 60px;
+              height: 60px;
+              margin: 10px 0 0 10px;
+              background-size: 40px;
```

Comment:
Shouldn't the style related to the "Add a file" plugin be in ECMS ?

Reply:
yess agree !